src/box/tuple.c

```diff
@@ -392,9 +392,6 @@ tuple_arena_create(struct slab_arena *arena, struct quota *quota,
 				       " tuple arena", prealloc, arena_name);
 		}
 	}
-
-	say_debug("tuple arena %s: addr %p size %zu flags %#x dontdump %d",
-		  arena_name, arena->arena, prealloc, flags, dontdump);
```

Comment:
It isn't okay to remove the logging code without an explanation. Can you leave it?

Reply:
As discussed f2f - deleted this log and tuple new/delete debug logs in distinct PR https://github.com/tarantool/tarantool/pull/9113